### ISSUE TITLE
Front end empty on iPhone and ipad

### ISSUE THREAD

**Issue by conradhagemans** [nurikk/z2m-frontend#1360]
On the desktop the front end loads fine. But in my mobile devices iPad and iPhone not. 
I get the header shown but the devices are not shown. 
is this a bug?
How to fix?

**Comment by steve-duff** [comment]
I don't think the "login" process works well at all. If you enter the wrong or no password at all you get stuck with an unauthorised session that you can't get rid of. There's no "logout"  option to get rid of the session. Also the fact that it isn't really a password entry also defeats my password manager.

It is a beta I suppose, and the workaround I found is to use a private browsing tab in Safari and enter your password again.

**Comment by JevVan** [comment]
I had a blank UI as well.  It started after the 2022.5 HA update.  So I spent a long time checking and changing my config thinking that would help but with no luck.  So I searched and ended up here.  

If I log in via a private window, the UI works.  So I deleted all cookies (best I can tell) associated with my Home assistant server.  It required me to log in, like in Private, but the UI is still a blank window?
